ott/core/initializers.py

```diff
@@ -0,0 +1,305 @@
+# Copyright 2022 The OTT Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sinkhorn initializers."""
+from typing import Optional, Tuple
+
+import jax
+import jax.numpy as jnp
+
+from ott.core import linear_problems
+from ott.geometry.pointcloud import PointCloud
```

Comment:
remark about consistency of imports (i.e. stop at geometry import pointcloud then pointcloud.PointCloud

Reply:
Changed